tests/components/uvc/test_camera.py

```diff
@@ -103,26 +133,32 @@ def test_setup_partial_config_v31x(self, mock_uvc, mock_remote):
         assert setup_component(self.hass, "camera", {"camera": config})
 
         assert mock_remote.call_count == 1
-        assert mock_remote.call_args == mock.call("foo", 7080, "secret", ssl=False)
-        mock_uvc.assert_has_calls(
-            [
-                mock.call(mock_remote.return_value, "one", "Front", "ubnt"),
-                mock.call(mock_remote.return_value, "two", "Back", "ubnt"),
-            ]
-        )
+        assert mock_remote.call_args == mock.call("foo",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)